### ISSUE TITLE
FPS

### ISSUE THREAD

**Issue by yanlongbinluck** [ZJULearning/ttfnet#20]
Hi，is the FPS computed by mmdetection as follow?
`./tools/dist_test.sh configs/ttfnet/ttfnet_d53_2x.py /path/to/the/checkpoint 1`
My gpu is 2080ti, but I can only get inferred speeds of 90FPS (90 task/s) for TTFnet18. 
What does ''using converged models'' mean?
thanks

**Comment by liuzili97** [comment]
No. The slower speed is because the time consumed by data loading is also included in your case. The reported FPS in our paper only includes the time consumed by the model.
Using converged models means that the final checkpoint file is used.

**Comment by yanlongbinluck** [comment]
I see. thanks

**Comment by 994374821** [comment]
> No. The slower speed is because the time consumed by data loading is also included in your case. The reported FPS in our paper only includes the time consumed by the model.
> Using converged models means that the final checkpoint file is used.
What is your batch size which test? 
DId your reported time include preprocess of input image ?
